README.md

```diff
@@ -1,16 +1,58 @@
-This directory contains the programs needed to generate and verify SigStore root keys and create signed TUF metadata. 
+This directory contains the programs needed to generate and verify Sigstore root keys and create signed TUF metadata. 
+
+## TUF Repository Structure
+
+The current published repository metadata lives in the [repository](/repository/repository) subfolder of this GitHub repository. In this repository, you will find the top-level TUF metadata files, delegations, and target files. 
+
+* [root.json](repository/repository/root.json): This is the current `root.json`. It is signed by at least 3 out of the 5 [current root keyholders](https://github.com/sigstore/root-signing#current-sigstore-root-keyholders). Other signing keys endorsed by this root include:
+  * A set of 5 targets keyholders (the root keyholders).
+  * An online [snapshotting key](https://github.com/sigstore/root-signing/blob/57ac5cd83b90ff97af78db846eea2525eb0eee81/repository/repository/root.json#L87-L97) located at `projects/project-rekor/locations/global/keyRings/sigstore-root/cryptoKeys/snapshot`. You can verify the hex-encoded public key value with the following code:
+  ```
+  $ gcloud kms keys versions get-public-key 1 --key snapshot --keyring sigstore-root --location global --project project-rekor | openssl ec -pubin -noout -text 
```

Comment:
Same comment here and below - This isn't usable by most folks, only those who maintain the public instance